src/utils/capmodpp.hpp

```diff
@@ -45,15 +42,17 @@ namespace capmodpp {
  * @warning If this is thrown then this usually means program is in insecure state and you should exit or abort (see above).
  * @style This does not inherit from std::exception, to make developers catch it explicitly if they want to.
  */
-class capmodpp_error {
+class capmodpp_error : public std::exception {
 	public:
```

Comment:
:orange_book:
Now we decided to do use this special exceptions.
I will commit docs on it soon